samples/properties_user_links_audit_test.py

```diff
@@ -0,0 +1,25 @@
+#
```

Comment:
It looks like the copyright year is missing in this file

Reply:
Done